lib/alex_koin/account/wallet.ex

```diff
@@ -4,12 +4,15 @@ defmodule AlexKoin.Account.Wallet do
   use Ecto.Schema
   import Ecto.Changeset
   alias AlexKoin.Account.User
+  alias AlexKoin.Coins.Coin
+  alias __MODULE__
 
 
   schema "wallets" do
     field :balance, :float
 
     belongs_to :user, User
+    has_many :coins, Coin, foreign_key: :mined_by_id
```

Comment:
This name seems a little weird; when you transfer a koin to another wallet, does this field get updated?

Reply:
Hrm... I do not. I'm trying to remember why I did this.